flink/README_streaming.md

```diff
@@ -101,11 +101,11 @@ In addition to supporting the configuration in [TiDB Batch Mode](./README.md), t
 | Configuration                          | Default Value | Description                                                                                                                              |
 |:---------------------------------------|:--------------|:-----------------------------------------------------------------------------------------------------------------------------------------|
 | tidb.streaming.source                  | -             | The data source(messaging system) where TiDB's change logs are stored, currently only supports Kafka and Pulsar will be supported later. |
-| tidb.streaming.codec                   | -             | TiDB's change log encoding method, currently supports default(json), craft, canal-json. See [TiDB Metadata](#7 Codec)                    |
+| tidb.streaming.codec                   | craft         | TiDB's change log encoding method, currently supports default(json), craft, canal-json. See [TiDB Metadata](#7 Codec)                    |
```

Comment:
Why description says that default(json)

Reply:
json named default in lower tidb version, and named json in higher tidb version.